rust-code-analysis-web/Cargo.toml

```diff
@@ -3,6 +3,8 @@ name = "rust-code-analysis-web"
 version = "0.0.18"
 authors = ["Calixte Denizet <cdenizet@mozilla.com>"]
 edition = "2018"
+documentation = "https://docs.rs/rust-code-analysis/"
```

Comment:
We don't have a documentation for the web crate yet, so perhaps we should remove this line